GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -79,10 +79,11 @@ public static ArgumentString AddSubmoduleCmd(string remotePath, string localPath
                 };
         }
 
-        public static ArgumentString GetRefsCmd(bool tags, bool branches, bool noLocks, GitRefsSortBy sortBy, GitRefsSortOrder sortOrder)
+        public static ArgumentString GetRefsCmd(GetRefsEnum getRef, bool noLocks, GitRefsSortBy sortBy, GitRefsSortOrder sortOrder)
         {
             string format;
-            if (!tags)
+            bool hasTags = (getRef & GetRefsEnum.All) != 0 || (getRef & GetRefsEnum.Tags) != 0;
```

Comment:
Did you miss this one?

Reply:
This is different - I can add a special if you prefer